packages/elements/src/chart/elements/chart.ts

```diff
@@ -402,11 +402,7 @@ export class Chart extends BasicElement {
         // These types, Colors could be string or array
```

Comment:
Is this comment still relevant to the new change ?

Reply:
updated